clarifai/client/model.py

```diff
@@ -90,36 +90,43 @@ def create_model_version(self, **kwargs) -> 'Model':
 
     return Model(**kwargs)
 
-  def list_versions(self) -> List['Model']:
+  def list_versions(self, page_no: int = None,
+                    per_page: int = None) -> Generator['Model', None, None]:
```

Comment:
This goes for all. What does `None` in `Generator['Model', None, None]` represent here, should we use `Union` here?!

Reply:
[This](https://docs.python.org/3/library/typing.html#:~:text=If%20your%20generator%20will%20only%20yield%20values%2C%20set%20the%20SendType%20and%20ReturnType%20to%20None%3A) is the reason it was kept as None. Since we're only yielding values, send_type and return_type are set to None.